apps_accessors_test.go

```diff
@@ -0,0 +1,2548 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Code generated by gen-accessors; DO NOT EDIT.
+// Instead, please run "go generate ./..." as described here:
+// https://github.com/google/go-github/blob/master/CONTRIBUTING.md#submitting-a-patch
+
+package godo
+
+import (
+	"testing"
+)
+
+func TestApp_GetActiveDeployment(tt *testing.T) {
```

Comment:
How are these tests checking for functionality? There looks to be no asserts or checks.

Reply:
These tests are checking for nil-safety for the most part i.e. no panics